packages/firestore/test/unit/local/local_store.test.ts

```diff
@@ -301,7 +301,12 @@ function genericLocalStoreTests(
     localStore = new LocalStore(persistence, User.UNAUTHENTICATED);
   });
 
-  afterEach(() => persistence.shutdown(/* deleteData= */ true));
+  afterEach(async () => {
+    await persistence.shutdown();
+    await IndexedDbPersistence.clearPersistence(
+      persistenceHelpers.TEST_PERSISTENCE_PREFIX
```

Comment:
Since we're repeating this phrase so often, maybe extract this into `persistenceHelpers.clearPersistence()`?